src/Backend/Routes/signIn.tsx

```diff
@@ -0,0 +1,24 @@
+// sign in
+import { signInWithEmailAndPassword } from "firebase/auth";
+import auth from "../index"
+
+
+async function signIn(email:string, password:string):Promise<boolean> {
+  const success = await signInWithEmailAndPassword(auth, email, password)
+  .then((userCredential) => {
+    // Signed in
+    const user = userCredential.user;
+    console.log(user)
```

Comment:
Probably return the user instead of a boolean and chuck an error otherwise instead of catching it